### ISSUE TITLE
Problems with code completion

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tsaiyilien/google-plugin-for-eclipse#42]
```
While coding and trying to use Eclipse Autocompletion facility, once the 
desired method is chosen and Enter key is pressed, an error arises (see 
attached file)

Ubuntu 11.04, Eclipse 3.5.2, Google plugin 2.5.0.v201112160242-rel-r35


```

Original issue reported on code.google.com by `JuanPabl...@gmail.com` on 2 Jan 2012 at 6:39

Attachments:
- [eclipse plugin error.png](https://storage.googleapis.com/google-code-attachments/google-plugin-for-eclipse/issue-42/comment-0/eclipse plugin error.png)

**Comment by GoogleCodeExporter** [comment]
```
Sorry, just realised I'd posted a duplicate issue in 
http://code.google.com/p/google-plugin-for-eclipse/issues/detail?id=44
```

Original comment by `wingnu...@gmail.com` on 10 Jan 2012 at 7:39

**Comment by GoogleCodeExporter** [comment]
```
We are working on this.
```

Original comment by `chintanr...@google.com` on 27 Feb 2012 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.5.2. please check.
```

Original comment by `chintanr...@google.com` on 7 Mar 2012 at 12:31
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Guys, you're great!! I checked and it works! Thanks!!
```

Original comment by `JuanPabl...@gmail.com` on 7 Mar 2012 at 2:14
